app/views/posts/show.html.erb

```diff
@@ -7,4 +7,27 @@
     <a class="btn btn-primary btn-sm" <%= link_to "Edit", edit_post_path(@post) %></a> 
     <%end%>
   </div> 
-</div>
\ No newline at end of file
+</div>
+
+<h4 class="mt-4">Comments</h4>
+<% if !@post.comments.blank? %>
+<% @post.comments.each do |comment| %>
+<div class="card mt-2">
+  <div class="card-header">
+    <div class="card-flex">
+      <div><%= gravatar_for comment.user , 40 %></div>
+      <div class="post-header">
+        <p class="post-header-detail"><%= link_to comment_owner(comment.user), comment.user%></p>
+        <p class="post-header-detail"><%= comment.user.email%></p>
+        <p class="post-header-detail"><%= time_ago_in_words(comment.created_at) %> ago</p>
+      </div>
+    </div>
+  </div>
+  <div class="card-body">
+    <p class="card-text"><%= comment.body%></p>
+  </div>
+</div>
```

Comment:
- [ ] Let's add post likes in post show page. 

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts4.png)